### ISSUE TITLE
How does mega discriminator lable "human-written disinformation"?

### ISSUE THREAD

**Issue by 7fantasysz** [rowanz/grover#23]
According to your blog, grover achieved great accuracy on both neural fake news as well as human-written disinformation. The output label of your released mega is ['machine', 'human']. So, how do you label the "human-written disinformation", it should be "human" literally but should be "machine" because it is "fake".

Your work is very interesting to me and I am hoping to apply your model in near future.

**Comment by rowanz** [comment]
We added a separate label for that. The code for that isn't currently in the repo, but it's a pretty simple change (just add the third label). best of luck!

**Comment by moonstarai** [comment]
Thanks for your comments. I'd like to reproduce your interesting and great work. Therefore, I am wondering whether you could share human-written disinformation data set you collected (as you mentioned in the blog). If not, could you please tell me how you collect these 30k data? Cheers!

**Comment by rowanz** [comment]
Sure, the raw data is here gs://grover-models/data/data_3way_human_prop_machine_june7.jsonl . lmk if that works. thanks!

**Comment by moonstarai** [comment]
Thanks so much for sharing the data. But I tried to download in the following two ways:
`gsutil cp gs://grover-models/data/data_3way_human_prop_machine_june7.jsonl .
wget https://storage.googleapis.com/grover-models/data/data_3way_human_prop_machine_june7.jsonl
`
Both reported 403 access denied error. Might you please check if you made this data file public accessible? Thank you!

**Comment by rowanz** [comment]
sorry, should be fixed now. thanks!

**Comment by moonstarai** [comment]
I downloaded it this time. Much appreciated for sharing!

**Comment by ajlane** [comment]
I've been trying to reproduce this. I added the extra label, and ran `run_discrimination.py` on a TPU pod against the 3-class dataset.

But the loss barely decreases before training ends, and the final model always predicts `human`.

```python
LABEL_LIST = ['machine', 'human', 'propaganda']
```

```bash
PYTHONPATH=${PWD} python discrimination/run_discrimination.py --input_data data/data_3way_human_prop_machine_june7.jsonl --output_dir gs://<my-bucket> --config_file lm/configs/large.json --do_train --tpu_name <my-tpu-pod> --tpu_zone <my-zone> --use_tpu
```

@rowanz Were there any other changes to the training code? How many epochs did you need? Do you still have the checkpoints for that model?
